kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -598,24 +598,19 @@ def exec_run(options: RunOptions) -> None:
         )
 
     if options.proof_hint:
-        kore_input = options.input_file.name.split('.')[0] + '.kore'
-        logging.warning('Proof hints are not supported in the pyk krun at the momment.')
-        logging.info('Saving the input file as %s', kore_input)
-        logging.info(
-            'If you want to emit hints take the krun command shown below and run it replacing the temp file by the kore file generated and append the `--proof-hint` flag to your command.'
+        logging.warning('Running KEVM with proof hint options enabled, remember to not use this with any `--verbose` mode if you\'re redirecting its output.')
+        output_bytes = kevm.run_proof_hint(kore_pattern, depth=options.depth, parser='cat', term=True, proof_hint=True)
```

Comment:
does the function we created not have a "debugger" or "expand_macros" flag?

Reply:
Yes, it has support for it! I've just added!